### PR TITLE
Release blog post for 4.3-beta2

### PR DESCRIPTION
**Pull request by akien-mga**
For social media:

TL;DR:
- A lot of critical bugs have been squashed, so this beta snapshot is more stable and a great entry point for new users to join the beta testing effort.
- This snapshot includes Adam's [audio samples](https://github.com/godotengine/godot/pull/91382) support for the Web export, solving issues of crackling audio in single-threaded mode. In other words, with this Godot 4.3's web export becomes much more suitable for exporting games to web platforms easily.

URL once published:
- https://godotengine.org/article/dev-snapshot-godot-4-3-beta-2/

Cover game:
- Stunt Xpress
- https://store.steampowered.com/app/2645830/Stunt_Xpress/
- Cover image: https://shared.akamai.steamstatic.com/store_item_assets/steam/apps/2645830/ss_efc105aa3d43ef7f0983c55a3e2970f9e00fa718.1920x1080.jpg?t=1718139120
- https://x.com/gonzelvis and https://x.com/pineapple_works

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-3-beta-2.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+excerpt: "Godot 4.3 is in beta, and progressing well towards stable. This beta 2 fixes many critical issues reported by testers of the previous snapshots."
+categories: ["pre-release"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/dev-snapshot-godot-4-3-beta-2.webp
+image_caption_title: "Stunt Xpress"
+image_caption_description: "A game by Rafael Gonçalves and Pineapple Works"
+date: 2024-06-20 11:30:00
+---
+
+We have been busy during the frist 3 weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-beta-2.md]:
```suggestion
We have been busy during the first three weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!
```

File: collections/_article/dev-snapshot-godot-4-3-beta-2.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+- Network: Crypto: Expose TLSOptions getters ([GH-93178](https://github.com/godotengine/godot/pull/93178)).
+- Particles: Fix collided 3D GPU particles sometimes jittering ([GH-92474](https://github.com/godotengine/godot/pull/92474)).
+- Porting: Make displayed Web errors more meaningful ([GH-92553](https://github.com/godotengine/godot/pull/92553)).
+- Porting: EGL: Use `EGL_EXT_platform_base` whenever possible ([GH-92663](https://github.com/godotengine/godot/pull/92663)).
+- Porting: Android: Fix invalid return value when multiple permission requests are dispatched ([GH-92709](https://github.com/godotengine/godot/pull/92709)).
+- Porting: macOS: Improve native menu open/close callbacks ([GH-92781](https://github.com/godotengine/godot/pull/92781)).
+- Porting: X11: Detect XWayland and disable screen capture support ([GH-93072](https://github.com/godotengine/godot/pull/93072)).
+- Porting: Windows: Add Windows version and Wine checks for Windows 10+ dark mode API ([GH-93126](https://github.com/godotengine/godot/pull/93126)).
+- Rendering: vulkan: Update all components to Vulkan SDK 1.3.283.0 ([GH-92010](https://github.com/godotengine/godot/pull/92010)).
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-beta-2.md]:
```suggestion
- Rendering: Vulkan: Update all components to Vulkan SDK 1.3.283.0 ([GH-92010](https://github.com/godotengine/godot/pull/92010)).
```

File: collections/_article/dev-snapshot-godot-4-3-beta-2.md (hunk 1 above)
Comment (coppolaemilio) [inline collections/_article/dev-snapshot-godot-4-3-beta-2.md]:
```suggestion
We have been busy during the first 3 weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!
```

Comment (coppolaemilio) [reply collections/_article/dev-snapshot-godot-4-3-beta-2.md]: better to use JohnVeness edit 👍
